org.eclipse.jdt.ls.tests/build.properties

```diff
@@ -3,7 +3,7 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                plugin.xml,\
-               plugin.properties
-src.includes = src/,\
-               projects/,\
+               plugin.properties,\
+               testresources/
```

Comment:
why do you need to include testresources in the binary?

Reply:
I'd rather prefer th rtstubs18.jar was moved to a fakejdk directory that can be added to the binary, and leave testresources out of it